.github/workflows/build-kata-static-tarball-s390x.yaml

```diff
@@ -20,6 +20,7 @@ jobs:
           - qemu
           - rootfs-image
           - rootfs-initrd
+          - se-image
```

Comment:
IIU the se-image is a repack of rootfs-initrd with some specific stuffs and generated as a ramdisk image. So could we call this new image as "rootfs-image-se" to follow the name pattern?

Reply:
Unfortunately, it is a repackage of kernel + rootfs-initrd. So it is a little brave to call it `rootfs-`.

Reply:
ah, I see... indeed calling it `rootfs` would be wrong. What about `boot-image-se`? According to the doc you pasted below and IIUC this is referred as "boot image". Sorry for the nit, it is that "image" can be anything :D

Reply:
It sounds much better. Wdyt, @hbrueckner ?